.github/workflows/integration-tests.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
       RSC_VERSION: 2022.09.0
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       RSC_LICENSE: ${{ secrets.RSC_LICENSE }}
```

Comment:
I updated this secret in GH so that it is now a file path where a license file can be expected, not a license key.

Reply:
In other contexts (Connect testing), we use `CONNECT_LICENSE` for a key and `CONNECT_LICENSE_FILE` for the file. That lets us understand if we should `license-manager activate` or copy the file to the landing place.

The Connect Docker images uses `RSC_LICENSE` for a key and `RSC_LICENSE_FILE_PATH` for file. That's probably what inspired this workflow. https://github.com/rstudio/rstudio-docker-products/blob/d075a3c0a7bd8efc6b679ae33eceaa905234c30b/connect/startup.sh#L32-L40

Reply:
Yeah I was going to rename a bunch of things once I got it working, I agree it's not ideal as it is.